### PR TITLE
feat: compare function added

### LINKED ISSUE #560: Train the meetup corpus with Django views (Backend)
## Description
Train the meetup corpus data and then integrate the tags in django views.

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
8 days

### PR DESCRIPTION
**Pull request by ritwickraj78**
### Description
Added compare function to generate similarity score between meetup descriptions

Fixes #560 

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Tried testing with meetup descriptions from meetup.com. Works well

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by ritwickraj78** [comment]
@sakshi1499 The not coming is removed in this PR. We can merge that one now

**Comment by sakshi1499** [comment]
Ok ok got it

**Review by sakshi1499 — COMMENTED** [review]
Some checks are failing can you correct them? @ritwickraj78

**Review by SanketDG — DISMISSED** [review]
This looks okay but you will have to break this function down into composable units of building blocks, instead of one large function.

**Comment by ritwickraj78** [comment]
@sakshi1499 @SanketDG Really sorry. Will make the changes asap

File: systers_portal/meetup/compare.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        query_doc_bow = dictionary.doc2bow(query_doc)
+        query_doc_tf_idf = tf_idf[query_doc_bow]
+        sum_of_sims = (np.sum(sims[query_doc_tf_idf], dtype=np.float32))
+        avg = sum_of_sims / len(sent1)
+        avg_simscore.append(avg)
+        total_avg = np.sum(avg_simscore, dtype=np.float)
+        percentage_of_similarity = round(float(total_avg) * 100)
+        if percentage_of_similarity >= 100:
+            percentage_of_similarity = 100
+    return percentage_of_similarity
```
Comment (sammy1997) [inline systers_portal/meetup/compare.py:59]: @ritwickraj78 Can you modularize this a bit? And maybe add some comments

Comment (ritwickraj78) [reply systers_portal/meetup/compare.py:59]: Sure.

**Review by sammy1997 — DISMISSED** [review]
@sakshi1499 @SanketDG Ritwick has made it a bit modular. The rest of it looks good to me. Let's wrap up the reviews and merge these. We are quite behind on these

**Comment by sammy1997** [comment]
@ritwickraj78 please resolve the merge conflicts

**Review by SanketDG — APPROVED** [review]

**Review by sakshi1499 — APPROVED** [review]

**Pull request merged**